packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -81,7 +81,7 @@ export class FakeChannelProvider implements ChannelProviderInterface {
   }
 
   setState(state: ChannelResult): void {
-    this.latestState = state;
+    this.latestState = {...this.latestState, [state.channelId]: state};
```

Comment:
Why is key in brackets? Shouldn't it be
```typescript
{...this.latestState, state.channelId: state};
```
Is this a typescript thing?

Reply:
I think that's a [ES2015 thing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names). If the key isn't a string literal, you have to put it in brackets.